docs/pages/access-controls/idps/saml.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Using Teleport as a SAML identity provider
+description: How to configure and use Teleport as a SAML identity provider.
+h1: Teleport as a SAML identity provider
+---
+
+## How to use Teleport as an identity provider
+
+This guide will cover how to use Teleport as an identity provider so that Teleport
+can be used to authenticate to external services. 
+
+## Prerequisites
+
+- Teleport role with access to maintaining `saml_idp_service_provider` resources.  This is available in the default `editor` role.
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Configuring the SAML identity provider service
+
+The SAML identity provider service is enabled by default on the proxy service
+for all Teleport users, so no configuration is necessary.
+
+## Establish a trust relationship between an application and Teleport
+
+You'll need to establish a trust relationship between an application, which will
+be functionining here as a **SAML service provider**, and Teleport's SAML identity
+provider. This process can vary greatly betwen applications, so you will likely
+need to refer to the target application's documentation in order to figure out how to
+add Teleport's identity provider metadata to the application.
+
+### Obtain the service provider metadata and add it to Teleport
+
+First, you must retrieve the service provider metadata in XML format from the
```

Comment:
Is there a well-known application that supports SAML we could use as an example?

Reply:
Perhaps https://sptest.iamshowcase.com ??

Reply:
I used `samltest.id`. Let me know what you think.